app/2.0/docs/browsers.md

```diff
@@ -38,7 +38,7 @@ td.polyfill {
 
 <table>
 <thead>
-  <tr><th></th><th>Chrome</th><th>Firefox</th><th>IE&nbsp;11+/<br>Edge</th><th>Safari 9+</th><th>Chrome
+  <tr><th></th><th>Chrome</th><th>Firefox</th><th>IE&nbsp;11+/<br>Edge</th><th>Opera</th><th>Safari 9+</th><th>Chrome
```

Comment:
Cell padding, whyyyy is the text in the Opera column slightly more smooshed? I could not figure this out, the markdown looks completely fine. Bah